### PR TITLE
[WIP] multicluster multi-network shared control plane update

### LINKED ISSUE #6594: Issues with https://preliminary.istio.io/docs/setup/install/multicluster/shared-gateways/
Need to update the list of pods for cluster 1:
```
NAME                                    READY   STATUS    RESTARTS   AGE
istio-ingressgateway-6767869fcf-sq7n9   1/1     Running   0          17s
istiod-7f7b4dc6d9-2xwmt                 1/1     Running   0          35s
prometheus-68d688f8b7-nmlrn             1/2     Running   0          16s
```

When updating IPs in the config map, we are down to only one instance of each. We mention two in the docs, but that was a 1.4 only thing I guess.

~~Fails trying to deploy remote profile on cluster~~ It worked the second time through

Wait for the Istio pods on cluster2, except for istio-ingressgateway, to become ready....
there is only an ingress gateway pod now, so not sure, but maybe remove this wait.

ingress-gateway fails to start with a:
```
MountVolume.SetUp failed for volume "citadel-ca-cert" : configmap "istio-ca-root-cert" not found
```
See https://github.com/istio/istio/issues/21281

### PR DESCRIPTION
**Pull request by linsun**
provide 1.5 update for this page:

https://preliminary.istio.io/docs/setup/install/multicluster/shared-gateways/#setup-cluster-1-primary

fixes #6594

### CODE REVIEW

File: content/en/docs/setup/install/multicluster/shared-gateways/index.md [hunk 1]
```diff
@@ -181,32 +176,19 @@ This will be used to access pilot on `cluster1` securely using the ingress gatew
...
-      --set values.security.selfSigned=false \
-      --set values.global.createRemoteSvcEndpoints=true \
-      --set values.global.remotePilotCreateSvcEndpoint=true \
-      --set values.global.remotePilotAddress=${LOCAL_GW_ADDR} \
-      --set values.global.remotePolicyAddress=${LOCAL_GW_ADDR} \
-      --set values.global.remoteTelemetryAddress=${LOCAL_GW_ADDR} \
       --set values.gateways.istio-ingressgateway.env.ISTIO_META_NETWORK="network2" \
-      --set values.global.network="network2" \
-      --set values.global.multiCluster.clusterName=${CLUSTER_NAME} \
+      --set values.global.network="network2" \
```
Comment (frankbu) [inline content/en/docs/setup/install/multicluster/shared-gateways/index.md]:
```suggestion
      --set values.global.network="network2" \
```

File: content/en/docs/setup/install/multicluster/shared-gateways/index.md [hunk 2]
```diff
@@ -181,32 +176,19 @@ This will be used to access pilot on `cluster1` securely using the ingress gatew
...
-      --set values.global.createRemoteSvcEndpoints=true \
-      --set values.global.remotePilotCreateSvcEndpoint=true \
-      --set values.global.remotePilotAddress=${LOCAL_GW_ADDR} \
-      --set values.global.remotePolicyAddress=${LOCAL_GW_ADDR} \
-      --set values.global.remoteTelemetryAddress=${LOCAL_GW_ADDR} \
       --set values.gateways.istio-ingressgateway.env.ISTIO_META_NETWORK="network2" \
-      --set values.global.network="network2" \
-      --set values.global.multiCluster.clusterName=${CLUSTER_NAME} \
+      --set values.global.network="network2" \                                      
+      --set values.global.multiCluster.clusterName=${CLUSTER_NAME}
```
Comment (frankbu) [inline content/en/docs/setup/install/multicluster/shared-gateways/index.md]:
```suggestion
      --set values.global.multiCluster.clusterName=${CLUSTER_NAME}
```

**Comment by frankbu** [comment]
This file is deleted in #6843

**Comment by ericvn** [comment]
Is this still needed with #6843?

**Comment by linsun** [comment]
close it

**Pull request closed without merging**